code/src/containers/tables/orders-table/components/orders-table-body/OrdersTableBody.tsx

```diff
@@ -84,14 +90,57 @@ const OrdersTableBody = ({ order, onStatusChange }: OrderTableBodyProps) => {
     </AppSelect>
   );
 
+  const handleIsPaidChange = () => {
+    !isPaid && onIsPaidChange(true);
+  };
+
+  const emailField = (
+    <AppTypography variant="caption" className="spa-order-table__body-email">
+      {email}
+    </AppTypography>
+  );
+
+  const notPaidOrderCheckbox =
+    orderStatus !== orderDeliveryStatuses.Canceled ? (
+      <AppTooltip
+        followCursor
+        titleTranslationKey="ordersTable.notpaid.tooltip"
+      >
+        <AppCheckbox
+          className="spa-order-table__body-checkbox"
+          onChange={handleIsPaidChange}
+        />
+      </AppTooltip>
+    ) : (
+      <AppTooltip
+        followCursor
+        titleTranslationKey="ordersTable.canceled.tooltip"
+      >
+        <AppCheckbox className="spa-order-table__body-checkbox" disabled />
+      </AppTooltip>
+    );
```

Comment:
difference between these components inside condition is insignificant. Maybe we can just make conditions for props?

Reply:
I tried to change like that but there I can see two issues:
-firstly - code becomes less readable
-stryker coverage decreaced from 100% to 80% because of one condition we resived two and for dofferent branches(

![image](https://github.com/user-attachments/assets/310795f7-5244-4018-b2eb-2aa76a95a5b0)
![image](https://github.com/user-attachments/assets/c1cf7d42-08fc-43d4-9662-cfafffb9992b)

Reply:
Okay. Let's leave it as it is